runtime/src/bank.rs

```diff
@@ -891,6 +896,33 @@ impl Bank {
                 if lock_res.is_ok()
                     && !hash_queue.check_hash_age(&tx.message().recent_blockhash, max_age)
                 {
+                    // Too old! Let's see if we're using the Nonce program...
+                    let maybe_nonce_ix = &tx.message.instructions[0]; // Could search here... at least tx.msg.program_ids()...
+                    let program_id = tx.message.account_keys[maybe_nonce_ix.program_id_index as usize];
+                    if nonce_program::check_id(&program_id) {
+                        if let Ok(nonce_instruction::NonceInstruction::Nonce) = limited_deserialize(&maybe_nonce_ix.data) {
+                            let accounts = self.rc.accounts.load_accounts(
```

Comment:
I think `self.rc.accounts.load_slow()` (bad name) would be better and more direct here